### PR TITLE
Chore:(Docs) Add TS version of Svelte snippet(get-started)

### PR DESCRIPTION
**Pull request by miily8310s**
Closes #

## What I did

Add TS version of Svelte snippet(only on get-started page)

## Checklist

- [x] Make sure your changes are tested (stories and/or unit, integration, or end-to-end tests)
- [x] Make sure to add/update documentation regarding your changes
- [ ] If you are deprecating/removing a feature, make sure to update
      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers

- [ ] If this PR should be tested against many or all sandboxes,
      make sure to add the `ci:merged` or `ci:daily` GH label to it.
- [ ] Make sure this PR contains **one** of the labels below.

`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`

### CODE REVIEW

File: docs/snippets/svelte/button-story-with-args.ts-4-9.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+/*
+ *👇 Render functions are a framework specific feature to allow you control on how the component renders.
+ * See https://storybook.js.org/docs/svelte/api/csf
+ * to learn how to use render functions.
+ */
+export const Primary: Story = {
+  render: (args) => ({
+    Component: Button,
+    props: args,
+  }),
```
Comment (jonniebigodes) [inline docs/snippets/svelte/button-story-with-args.ts-4-9.mdx:29]: @miily8310s, we can drop the render function and comment in this example and the pre-TS variation with the same name.

File: docs/snippets/svelte/histogram-story.ts-4-9.mdx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+/*
+ *👇 Render functions are a framework specific feature to allow you control on how the component renders.
+ * See https://storybook.js.org/docs/svelte/api/csf
+ * to learn how to use render functions.
+ */
+export const Default: Story = {
+  render: (args) => ({
+    Component: Histogram,
+    props: args,
+  }),
```
Comment (jonniebigodes) [inline docs/snippets/svelte/histogram-story.ts-4-9.mdx:29]: @miily8310s the same applies here. We can drop the render function and comment for this and the pre-TS-4.9 variation

File: docs/snippets/svelte/your-component.ts-4-9.mdx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+/*
+ *👇 Render functions are a framework specific feature to allow you control on how the component renders.
+ * See https://storybook.js.org/docs/svelte/api/csf
+ * to learn how to use render functions.
+ */
+export const FirstStory: Story = {
+  render: (args) => ({
+    Component: YourComponent,
+    props: args,
+  }),
```
Comment (jonniebigodes) [inline docs/snippets/svelte/your-component.ts-4-9.mdx:30]: @miily8310s same rule applies here and for the follow-up pre-TS-4.9 variation

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@miily8310s, thanks for taking the time to put together this pull request and complement our existing examples. Appreciate it greatly 🙏 ! I left a couple of items for you to look at when you have a chance. For context, I'm aware that the existing Javascript variations for Svelte mirroring this pull request include explicit `render` functions, but for these examples, we don't need them, and if you're ok we can adjust them on a follow-up pull request. Would you be interested in it?

Let me know, and we'll go from there.

Looking forward to hearing from you and getting this one in.

Have a great day.

Stay safe

**Comment by miily8310s** [comment]
> if you're ok we can adjust them on a follow-up pull request. Would you be interested in it?

OK

**Comment by jonniebigodes** [comment]
@miily8310s, that's great to hear. Glad that you're willing to do it 🙏 ! 

Let me know once you've addressed the feedback so that we can get this one in and go from here for the next one.

Once again, thanks for this great contribution ❤️

**Comment by miily8310s** [comment]
@jonniebigodes 

> Let me know once you've addressed the feedback so that we can get this one in and go from here for the next one.

I'm sorry for responding slowly. I'd reflected your feedback, please check my modification!

**Comment by jonniebigodes** [comment]
@miily8310s, no need to be sorry whatsoever. I understand it. I checked the changes, and they're great. Once again thanks for addressing the feedback so promptly. Appreciate it 🙏 ! I'm going to go ahead and merge this one. And if you're interested in contributing to the changes for the JS versions, let me know once the pull request is up, and I'll be more than glad to follow up with you and we'll go from there.

Hope you have a great week.

Stay safe

**Pull request merged**